updates/testing.json

```diff
@@ -47,6 +47,9 @@
     {
       "version": "33.20201116.2.0",
       "metadata": {
+        "barrier": {                                                                                                                                                                                                                         
+          "reason": "https://github.com/coreos/fedora-coreos-tracker/issues/646"                                                                                                                                                             
+        },
```

Comment:
Looks like there's a lot of extra padding here.

Reply:
fixed the padding and rebased.